index.html

```diff
@@ -1229,6 +1232,19 @@ <h2>
 operations (see Section <a href="#did-operations"></a>), which can be
 defined by <a>DID method</a> specifications.
       </p>
+
+      <p class="note" title="Verification methods">
+A public key is just one type of <a>verification method</a>. Ideally,
+public keys are expressed in a <a>DID document</a> according to what
+they are authorized for: <code>authentication</code>, <code>
+capabilityInvocation</code>, <code>capabilityDelegation</code>,
+<code>keyAgreement</code>, <code>assertionMethod</code>, and <em>not</em>
+just listed using the <code>publicKey</code> property. The latter
```

Comment:
To clarify is the implication of the following sentence starting with "The latter....", that by default any keys listed under the public key section of the did document and not mentioned under any verification method, implying those keys should be interpreted as being authorised for all proof purposes? 

My understanding was that keys listed under the public key section of the did document were just expressions of association of keys to the did subject. Of course applications consuming the did document could choose to infer whatever authority they want from keys listed in the public key section however I don't think the specification should imply this ambient authority type behaviour as a default when no proof purposes are defined in the did document.

Reply:
Yes, agreed... boo to ambient authority, will fix.

Reply:
Fixed in: https://github.com/w3c/did-core/pull/123/commits/1f6cfc764b60af06396e0a88a3ac0ce084f1d154